modules/apps/transfer/types/transfer_authorization.go

```diff
@@ -9,10 +13,11 @@ import (
 	host "github.com/cosmos/ibc-go/v7/modules/core/24-host"
 )
 
-const gasCostPerIteration = uint64(10)
```

Comment:
This diff wasn't included in this backport but I've changed to use `ctx.KVGasConfig().IterNextCostFlat` to align with main.